client/webserver/site/src/js/orderutil.ts

```diff
@@ -123,9 +123,19 @@ export function matchStatusString (status: number, side: number) {
 // life easier.
 let orderOptTmpl: HTMLElement, booleanOptTmpl: HTMLElement, rangeOptTmpl: HTMLElement
 
-// setOptionTemplates sets the package vars for the templates and application.
-export function setOptionTemplates (page: Record<string, PageElement>) {
-  [booleanOptTmpl, rangeOptTmpl, orderOptTmpl] = [page.booleanOptTmpl, page.rangeOptTmpl, page.orderOptTmpl]
+// setRangeOptionTempalate sets the package range option template variable.
+export function setRangeOptionTempalate (tmpl: PageElement) {
```

Comment:
I changed this because on the `markets.tmpl` we had two instances of the `rangeOptTmpl`, one in the acceleration popup and one on the verify order popup. By doing this we can avoid having two elements with the same ID on the page.